src/Client/Extensions/StringExtensions.cs

```diff
@@ -0,0 +1,14 @@
+﻿namespace InterviewApp.Client.Extensions;
+
+public static class StringExtensions
+{
+    public static bool IsNullOrEmpty(this string value)
+    {
+        return string.IsNullOrEmpty(value);
+    }
+
+    public static bool IsNullOrWhiteSpace(this string? value)
+    {
+        return string.IsNullOrWhiteSpace(value);
```

Comment:
Expression body method

Reply:
I'm not too sure if
```cs
myString.IsNullOrWhiteSpace();
```

is that much better than

```cs
string.IsNullOrWhiteSpace(myString);
```

Especially since it's such a commonly used expression. 🤔 

My view is that we're always going to save 6 characters (for the `string` keyword) and I'm not sure if that's quite as worth it. 🤔 

I can also see future developers not knowing about this extension, and using `string.IsNullOrWhiteSpace` as usual. This might eventually cause the code to get unnecessarily complex to save 6 characters every call.

It is a pretty nice idea, and I of course appreciate you trying to improve readability. It's quite clever, I must add. But in the long run, I personally think this ain't it.

Though I'd love to hear your thoughts. ☺️ 💯